### DISCUSSION TITLE
Consider pop-up closing behavior

### DISCUSSION THREAD

**Discussion by geealbers** [thegetty/quire#779] [Feedback]
From an editor looking at the Mummy Portraits book.

> I notice that pop-up boxes remain open unless you reclick the highlighted text. This is true even when you have gone and opened other pop-up boxes around it (and sometimes the ones you click later open underneath one you opened earlier). Can’t help wondering if there is a way to make pop-up boxes close either upon reclicking highlighted text or upon opening a different pop-up nearby? Perhaps this is an improvement to implement in a later Quire book if it is more complicated than it sounds.

Currently, pop-ups can be opened in multiple. Would be worth looking at what the ideal behavior would be, particularly when we're also looking at web accessibility for the pop-ups.

**Comment by ronvoluted** [comment]
For web accessibility, the general checklist for closing pop-ups/modals is:
- Clicking/tapping outside the element
- Hitting a clickable + keyboard navigable close button
- Hitting the Escape key

**Comment by geealbers** [comment]
Awesome, thanks @ronvoluted!

**Comment by Erin-Cecele** [comment]
@geealbers, I think we should open this as a fresh issue to implement @ronvoluted's suggestion for web accessibility best practices. Which is more of an improvement than a bug, so we should discuss how best to handle workflow-wise. It also feels beneficial to keep this as a discussion for both visibility and further input.
